frame/bounties/src/tests.rs

```diff
@@ -1087,8 +1087,8 @@ fn accept_curator_handles_different_deposit_calculations() {
 		assert_ok!(Bounties::propose_bounty(Origin::signed(0), value, b"12345".to_vec()));
 		assert_ok!(Bounties::approve_bounty(Origin::root(), bounty_index));
 
-		System::set_block_number(3);
-		<Treasury as OnInitialize<u64>>::on_initialize(3);
+		System::set_block_number(4);
```

Comment:
The test code was wrong because the `on_initialize` hook of pallet-treasury will only fund bounties for even block numbers (at least in the test configuration).